Library/Formula/scirius.rb

```diff
@@ -0,0 +1,49 @@
+class Scirius < Formula
+  desc "Rules manager for Suricata IDPS"
+  homepage "https://github.com/StamusNetworks/scirius"
+  url "https://github.com/StamusNetworks/scirius"
```

Comment:
thanx! Sorry, I'm new to git that's why so many questions... I'll aske the owner of `scirius` about the specific release, like `https://github.com/StamusNetworks/scirius/archive/scirius-1.0.tar.gz` Or would you suggest to fork it and make it myself?